soh/soh/SaveManager.cpp

```diff
@@ -1651,7 +1675,11 @@ void SaveManager::LoadBaseVersion4() {
     SaveManager::Instance->LoadArray("randomizerInf", ARRAY_COUNT(gSaveContext.randomizerInf), [](size_t i) {
         SaveManager::Instance->LoadData("", gSaveContext.randomizerInf[i]);
     });
-    SaveManager::Instance->LoadData("isMasterQuest", gSaveContext.isMasterQuest);
+    int isMQ = 0;
```

Comment:
I can't recall how we were supposed to treat changes like this, is it the correct behavior to make this change in LoadV1, LoadV2, LoadV3, and LoadV4? I think so?

Reply:
I think so as if this change wasn't made, older save files would not migrate their rando/master quest status correctly.

Reply:
I'm pretty sure that's correct. I think we discussed having it be chained loading so loading V1 would mean V1 -> V2, V2 -> V3, V3 -> V4 but decided against it. Instead we have "this is how you load this old version of a save into the current save format"

Reply:
Honestly, I feel like maybe the chain load would have been better in the end, but the only other option would be to make a new loader version and settings migrator, which would be overkill for a single thing like this. I'm OK with this method of handling it.